servers/republik/graphql/schema-types.js

```diff
@@ -236,4 +238,23 @@ type MembershipPeriodStatsDay {
   cancelCount: Int!
   prolongCount: Int!
 }
+
+type StatementUserConnection {
+  totalCount: Int!
+  pageInfo: PageInfo
+  nodes: [StatementUser!]!
+}
+type StatementUser {
+  id: ID!
+  name: String!
+  username: String
```

Comment:
Maybe use `slug` here too?

Reply:
good catch :+1:

Reply:
I reverted it, because otherwise the current FE would break during deploy.